pkg/controller/certificates/requestmanager/requestmanager_controller.go

```diff
@@ -79,6 +87,7 @@ func NewController(
 	recorder record.EventRecorder,
 	clock clock.Clock,
 	certificateControllerOptions controllerpkg.CertificateOptions,
+	fieldManager string,
```

Comment:
Suggestion: remove this as the 'apply' functionality that would use this field manager is removed in a later commit

Reply:
See above